tools/examples/debugger_script/try_reproducibility.py

```diff
@@ -0,0 +1,17 @@
+import os
```

Comment:
What is the status of this script ?

Reply:
This script still leads to nested joblib parallel loops, which I think is not a good idea for testing reproducibility. 
I would suggest removing the parallelization in this script.

Reply:
The example should be running in parallel and produce the same result. 
It's a tools for running the same example in parallel multiple time in order to check if there some difference.

It's not a test. For testing, you should look the PR #345.

Reply:
How do you want to check the difference of two runs of the same example in this case?

As remark, I use this parallel loop because it's the loop used in sphinx gallery for the generation of documentation based on examples. This is a way to recreate the environment during the generation of documentation.

Reply:
No, this is not the case.
I use parallelisation to check if the value of multiple runs of the same example gives exactly the same result. It's a way to debug it.

I don't do only 2 times because I find that, in some cases, the error is trickier, and 2 runs is not enough for detecting it.